exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/columnreaders/VarLenColumnBulkInput.java

```diff
@@ -479,6 +483,17 @@ private void deinitOverflowData() {
     fieldOverflowStateContainer = null;
   }
 
+  private void checkCancellation() {
```

Comment:
move to a common utility class. Make it static.

Reply:
within Parquet or under exec/util?